testing/jormungandr-testing-utils/resources/explorer/graphql/schema.graphql

```diff
@@ -188,25 +202,35 @@ scalar PublicKey
 
 type Query {
   block(id: String!): Block!
-  blockByChainLength(length: ChainLength!): Block
-
-  """query all the blocks in a paginated view"""
-  allBlocks(first: Int, last: Int, before: IndexCursor, after: IndexCursor): BlockConnection!
+  blocksByChainLength(length: ChainLength!): [Block!]!
   transaction(id: String!): Transaction!
+
+  """get all current tips, sorted (descending) by their length"""
+  tips: [Branch!]!
```

Comment:
So this should be `branches` to be consistent with the node protocol's terminology...